framework/content/ecl-typography/ecl-typography-lists/_lists.scss

```diff
@@ -1,22 +1,7 @@
-ul:not(.listing, .social-media-links__list, .file__translations-list, .site-switcher__list, .menu, .site-user-menu__list, #admin-menu-wrapper ul),
-ol {
-  max-width: 32em;
```

Comment:
For the moment, I would remove this rule, you can add a comment that it's to be removed or that's it's legacy for backwards compatibility or something. This rule 32em is actually quite important for them and has to be handled carefully.